src/main/routes/routes.ts

```diff
@@ -282,6 +282,17 @@ export default function (app: Application): void {
         isPermittedMedia,
         app.locals.container.cradle.subscriptionManagementController.get
     );
+    app.get('/subscription-add-list', isPermittedMedia, app.locals.container.cradle.subscriptionAddListController.get);
```

Comment:
We need to add new a11y tests for the new routes too

Reply:
added.